compiler/x/codegen/SIMDTreeEvaluator.cpp

```diff
@@ -0,0 +1,237 @@
+/*
+ * Licensed Materials - Property of IBM
+ * "Restricted Materials of IBM"
+ *
+ * (c) Copyright IBM Corp. 2000, 2016 All Rights Reserved
+ *
+ * US Government Users Restricted Rights - Use, duplication or disclosure
+ * restricted by GSA ADP Schedule Contract with IBM Corp.
+ *
+ * This file is part of product 5655-W32.
+ *
+ */
+
+#include <stdint.h>                                   // for uint8_t, etc
+#include <stdio.h>                                    // for NULL, printf, etc
+#include "codegen/CodeGenerator.hpp"                  // for CodeGenerator, etc
```

Comment:
I'm not sure where this list of includes was copied from but it probably deserves evaluation for trimming since I don't think all of these entries are actually used in this file and so the compiler is working harder than needed and the code dependencies are less clear than they could be with at trimmed include list.